examples/using-mongodb/src/templates/item.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import Link from "gatsby-link"
+
+class Item extends React.Component {
+  render() {
+    console.log(this.props)
+    const story = this.props.data.mongoDbDocField;
+  
+    return (
+      <div>
```

Comment:
Looks like there's some inconsistent formatting. We use Prettier in this repo. Just run `npm run format` from the root of the repo to fix things up.